src/deployment/deployment-hapi-plugin.ts

```diff
@@ -146,20 +144,19 @@ class DeploymentHapiPlugin {
     }, {
       method: 'GET',
       path: '/raw-deployment-handler/{param*}',
-      handler: this.directoryHandler('key'),
+      handler: this.directoryHandler(PREKEY),
       config: {
         auth: 'customAuthorize',
-        pre: this.pre(auth, true, 'key'),
+        pre: this.pre(auth, true, PREKEY),
       },
     },
     {
       method: 'GET',
       path: '/deployments/{shortId}-{projectId}-{deploymentId}/{path*}',
-      handler: this.directoryHandler('key'),
+      handler: this.directoryHandler(PREKEY),
       config: {
-        pre: this.pre(auth, false, 'key'),
+        pre: this.pre(false, false, PREKEY), // Is authorized on a higher level
```

Comment:
All the routes that have either `projectId` or `teamId` parameters in their paths get authorized by [`AuthenticationHapiPlugin.authorize`](https://github.com/lucified/minard-backend/blob/redirect/src/authentication/authentication-hapi-plugin.ts#L355)